### PR TITLE
adds some tests for the universe contracts and fixes one bug in solid…

### PR DESCRIPTION
**Pull request by huesforalice**
…ity code

### CODE REVIEW

**Comment by huesforalice** [comment]
@rudolfix I have two questions:
"should set singleton without convenience method" -> I'm not sure what to test here exactly
"should remove interface from collection" -> This, I think, is now tested in the tests above, please let me know if you see this differently.

**Comment by rudolfix** [comment]
Hey did you upgrade linter? solium? The advice to always provide message for require is very bad. it costs a lot and contracts grow very quickly.

also javascript linter fails

File: contracts/Universe.sol [hunk 1]
```diff
@@ -175,6 +175,7 @@ contract Universe is
         uint256 idx;
         while(idx < interfaceIds.length) {
             setCollectionPrivate(interfaceIds[idx], instance, set);
+            idx += 1;
```
Comment (rudolfix) [inline contracts/Universe.sol]: cool!

File: test/Universe.js [hunk 2]
```diff
@@ -6,271 +6,542 @@ import registerSingletons from "./helpers/registerSingletons";
...
+        "platformTerms",
+      ];
+      for (let ii = 0; ii < kis.length; ii += 1) {
+        const loopi = await universe[getters[ii]]();
+        expect(loopi).to.eq(knownInstances[ii]);
+      }
+    });
 
-  it("should set singleton without convenience method");
+    it("should set singleton without convenience method");
```
Comment (rudolfix) [inline test/Universe.js]: the point of the test was to set a singleton that does not have convenience method (like universe.neumark()). a singleton where interface is just a random 4 bytes (not a knownInterface)

File: test/Universe.js [hunk 3]
```diff
@@ -6,271 +6,542 @@ import registerSingletons from "./helpers/registerSingletons";
...
-      knownInterfaces.tokenExchangeRateOracle,
-      knownInterfaces.identityRegistry,
-      knownInterfaces.commitmentInterface,
-    ]);
-    // remove from collection
-    await universe.setCollectionInterface(knownInterfaces.commitmentInterface, addr, false, {
-      from: universeManager,
+
+    // i would say this is tested above
+    it("should remove interface from collection");
```
Comment (rudolfix) [inline test/Universe.js]: yeah please remove

Comment (huesforalice) [reply test/Universe.js]: done

File: test/Universe.js (hunk 2 above)
Comment (huesforalice) [reply test/Universe.js]: done

**Comment by huesforalice** [comment]
@rudolfix format fix is run, also I have reverted back to an older version of the yarn lock file, I hope we won't get those error messages any more

**Pull request merged**